Description: Correct init script directives
 Correct values for the Default-Stop and Default-Start directives.
 Pulled from from http://http.debian.net/debian/pool/main/i/ibacm/ibacm_1.2.0-1.dsc
Author: Brian T. Smith <bsmith@systemfabricworks.com>
Forwarded: not required
Last-Update: <2017-12-12>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/ibacm.init.in
+++ b/ibacm.init.in
@@ -9,8 +9,8 @@
 # Provides:       ibacm
 # Default-Start: 2 3 4 5
 # Default-Stop: 0 1 6
-# Required-Start: @rdmascript@ $network
-# Required-Stop: @rdmascript@ $network
+# Required-Start: $network $remote_fs
+# Required-Stop: $network $remote_fs
 # Should-Start:
 # Should-Stop:
 # Short-Description: Starts and stops the InfiniBand ACM service
